glotaran/project/scheme.py

```diff
@@ -36,6 +36,7 @@ class Scheme:
     group_tolerance: float = 0.0
     non_negative_least_squares: bool = False
     maximum_number_function_evaluations: int = None
+    add_svd: bool = True
```

Comment:
I'd like this to be False by default, but this requires adapting tests, so maybe in a next PR.